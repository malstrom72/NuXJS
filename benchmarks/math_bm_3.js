function test(n) {
  var a = [];
  for (var i = 0; i < n; ++i) {
    a[i] = Math.sin(Math.PI * i / 10000.0) / 3.0;
  }
  var d = 0.0;
  for (var i = 0; i < n; ++i) {
    d += Math.pow(a[i] - a[n - 1 - i], 2);
  }
  return d;
}
print(test(1000000));
