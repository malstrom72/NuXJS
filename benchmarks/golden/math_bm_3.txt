222222.21673916807
